freezeyt_blog/app.py

```diff
@@ -5,22 +5,27 @@
 app = Flask(__name__)
 
 base_path = Path(__file__).parent
+content_path = base_path / 'content/'
+
+ARTICLES_PATH = content_path / 'articles/'
+STATIC_PATH = content_path / 'images/'
 
 @app.route('/')
 def index():
     """Start page with list of articles."""
+    post_names = [a.name.strip('.md') for a in sorted(ARTICLES_PATH.glob('**/*.md'))]
```

Comment:
The glob pattern **/*.md will find all .md files, even in subdirectories. Why do you want that?
Why not just *.md?

Reply:
if we will decide in future for more categories, where every single category will represent subdir in dir articles. But this idea is probably useless for our purpose.

Reply:
It is not useless, but it will *currently* not work: `@app.route('/<slug>')` does not allow directories (which is good for security, as I talked about on Monday).
If we want a feature like categories, we should actually create a category and test that it works. Without that, this is just making the code more complicated – and more complicated code has a bigger chance of having bugs.

Also note that [`strip`](https://docs.python.org/3/library/stdtypes.html?str.strip) takes a *set* of characters to be removed from *both* sides of the string: try e.g. `'my3d.md'.strip('.md')`. Using `stem` is much better here.

Reply:
I decide for strip beacuse arg 'name' return a 'string' so stem doesnot work. Ok I make simple code.

Reply:
Yah, I've understood now. I didn't read pathlib doc part about stem correctly. I supposed that `stem` just delete suffix not whole path.

I remove subdir searching and strip method.